template/module/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © <YEAR(s)> <AUTHOR(s)>
```

Comment:
Can you place the Author and Order lines in the same order as in the `__openerp__.py`?